pupil-spa/Dockerfile

```diff
@@ -68,4 +68,18 @@ COPY --from=builder /ng-app/docker-start.sh /
 #  * app.component.ts:9
 #RUN rm -rf /usr/share/nginx/html/assets
 
+# install ssh server
+RUN apk add openssh openrc \
+     && echo "root:Docker!" | chpasswd
+
+COPY sshd_config /etc/ssh/
+
+# auto start ssh on startup
+# RUN rc-update add sshd
```

Comment:
Is this comment useful?

Reply:
yes, im waiting on confirmation about how to integrate this in alpine, as currently the documented alpine best practice does not work, nor does microsofts approach.

Reply:
Aha

I was wondering whether I should point out that this is not the same as every other docker container and whether it wouldn't make more sense to use the same base image for ease of use and configuration.

Reply:
yes, we can discuss.  I would prefer to move away from containers completely, as its less baggage to maintain.  Will see what MS come up with on the support call